### ISSUE TITLE
https does not work

### ISSUE THREAD

**Issue by joukio** [ThomDietrich/home-assistant-addons#22]
hi,

I have home assistant running with https configured in configuration.yaml and the curl command will failed silently on that. Haven't got the time yet to debug further and come up with a solution, but wanted to mention it at least.

regards
Jouk

**Comment by ThomDietrich** [comment]
May I just ask WHY? :)

I wonder if the issue is with a self-signed or unverified certificate. Could you please do some testing? You should be able to produce a "200" status code with the following command (replace your IP address):

```
curl --write-out %{http_code} --silent --output /dev/null 192.168.23.211:8123
```

I suspect you will see a number other than 200. What happens if you add `--insecure` to the command?

**Comment by joukio** [comment]
certificate is from letsencrypt. Below some tests:

```
d8db2fc3e367:/# curl --write-out %{http_code} --silent --output /dev/null 172.30.32.1:8123
000d8db2fc3e367:/# echo $?
7
d8db2fc3e367:/# curl --write-out %{http_code} --insecure --silent --output /dev/null 172.30.32.1:8123
000d8db2fc3e367:echo $?
7
d8db2fc3e367:/# curl --write-out %{http_code} --silent --output /dev/null 172.30.32.1:443
000d8db2fc3e367:echo $?
52
d8db2fc3e367:/# curl --write-out %{http_code} --insecure --silent --output /dev/null 172.30.32.1:443
000d8db2fc3e367:echo $?
52
d8db2fc3e367:/#
```

**Comment by ThomDietrich** [comment]
I wonder if it would be better to move let's encrypt to your remote server. I suspect that you already have other subdomains hosted there and why would you bother to manage certificates at the end node? But we don't have to discuss this, you might have your reasons and others might have other reasons as well.

Please test again. I don't know what you did but 7 or 52 are no http status codes.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

**Comment by ThomDietrich** [comment]
Hey, your issue should be resolved with 1.3.3

**Comment by joukio** [comment]
on holiday right now, will take a look when @home again. Thnx in advance
